examples/consul-ami/README.md

```diff
@@ -12,7 +12,7 @@ These AMIs will have [Consul](https://www.consul.io/) installed and configured t
 boot-up. They also have [Dnsmasq](http://www.thekelleys.org.uk/dnsmasq/doc.html) installed and configured to use 
 Consul for DNS lookups of the `.consul` domain (e.g. `foo.service.consul`) (see [registering 
 services](https://www.consul.io/intro/getting-started/services.html) for instructions on how to register your services
-in Consul). To see how to deploy this AMI, check out the [consul-cluster example](https://github.com/hashicorp/terraform-aws-consul/tree/master/MAIN.md). 
+in Consul). To see how to deploy this AMI, check out the [consul-cluster](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules/consul-cluster). 
```

Comment:
Thanks for fixing these broken links! 

However, removing `example` and `module` from some of these texts might be misleading!

Reply:
I assume all the links' text should be `[consul-cluster module]` to be clear? brb

Reply:
See above, this should point to the root example. The same for most links below too.